.circleci/config.yml

```diff
@@ -981,6 +982,12 @@ workflows:
             - be-tests-ee
           driver: bigquery
 
+      - test-driver:
+          name: be-tests-bigquery-cloud-sdk-ee
```

Comment:
why not just update the existing BigQuery driver to use the Cloud SDK? We don't need to make a completely new driver every time the underlying dependencies change

Reply:
It's more or less a complete rewrite, and the general consensus is this introduces a lot of risk.  So Reza wants it to go through the deprecation cycle.  Besides that, it's dropping support for the non-service-account auth mechanisms entirely (not even work on upgrade), so it's not backward compatible.

Reply:
This has been rebased and now uses the deprecation logic from #17028.